docs/community/contributor_ladder.mdx

```diff
@@ -0,0 +1,178 @@
+* [Inactivity](#inactivity)
+* [Involuntary Removal](#involuntary-removal-or-demotion)
+* [Stepping Down/Emeritus Process](#stepping-downemeritus-process)
+* [Contact](#contact)
+
+
+## Contributor Ladder
+
+Hello! We are excited that you want to learn more about our project contributor ladder! 
+This contributor ladder outlines the different contributor roles within the project, 
+along with the responsibilities and privileges that come with them. 
+Community members generally start at the first levels of the "ladder" and advance up it as their involvement in the project grows.  
+Our project members are happy to help you advance along the contributor ladder.
+
+Each of the contributor roles below is organized into lists of three types of things. 
+"Responsibilities" are things that a contributor is expected to do. 
+"Requirements" are qualifications a person needs to meet to be in that role, 
+and "Privileges" are things contributors on that level are entitled to.
+
+### Contributor
+A Contributor contributes directly to the project and adds value to it. 
+Contributions need not be code. 
+People at the Contributor level may be new contributors, 
+or they may only contribute occasionally.
+
+* Responsibilities include:
+    * Follow the [CNCF CoC](https://github.com/cncf/foundation/blob/master/code-of-conduct.md)
+    * Follow the project [contributing guide](./contributing_guide.mdx)
+* Requirements (one or several of the below):
+    * Report and sometimes resolve issues
+    * Occasionally submit PRs
+    * Contribute to the documentation
+    * Show up at meetings, take notes
+    * Answer questions from other community members
+    * Submit feedback on issues and PRs
+    * Test releases and patches and submit reviews
+    * Run or helps run events
+    * Promote the project in public
+    * Help run the project infrastructure
+* Privileges:
+    * Invitations to contributor events
+    * Eligible to become an Organization Member
+
+### Owners Structure
+There are two types of owners in the ORAS project organized hierarchically. 
+ORAS org owners oversee the overall project and its health. 
+Subproject owners focus on a single repository, 
+a group of related repositories, a service (e.g., a website), 
+or subproject to support the other subprojects (e.g., marketing or community management).
+
+Changes in Oras Org owners have to be announced via an oras-project/community/issues. 
+Changes to sub-project owners are to be announced via the appropriate sub-project issue.
+
+### Owner Responsibility
+ORAS owners adhere to the requirements and responsibilities set forth in the respective ORAS Org Owners and Subproject Owners. 
+They further pledge the following:
+
+* To act in the best interest of the project and subprojects
+* To ensure that project and subproject development and direction is a function of community needs
+* To take action when reasonably confident that it is the right action to take
```

Comment:
This is a somewhat ambitious statement. There needs to be a concrete criteria that should be followed